api/src/main/resources/workbench-api.yaml

```diff
@@ -314,6 +314,27 @@ paths:
           description: Welcome Email sent.
         500:
           description: Internal Server Error
+  "/v1/send-billing-setup-email":
+    post:
+      tags:
+        - profile
+      consumes:
+        - application/json
+      description: Send billing setup email to GCP partner
+      operationId: sendBillingSetupEmail
+      security: [ ]
```

Comment:
careful, this makes the endpoint unauthenticated, should remove and take the default security setting

Reply:
good catch! Thank you